src/github/GithubClient.ts

```diff
@@ -79,17 +81,22 @@ export function dashWrap(str: string): string {
 
 /**
  * Logs all objects passed in debug outputting strings directly and
- * calling JSON.stringify on other elements
+ * calling JSON.stringify on other elements in a group with the given label
  */
-export function debugLog(...args: unknown[]): void {
+export function debugInspect(label: string, ...args: unknown[]): void {
```

Comment:
this makes the debugging log much easier to deal with by collapsing large json output into single lines in order to inspect the one or two someone may be interested in